momentum/utils.py

```diff
@@ -220,7 +220,25 @@ def rebalance_portfolio(universe, ranking_table, config, data):
     ignore_cols = ["TOTAL", "CASH", "PORTFOLIO"]
     sell = []
     zeros = []
-    liquidity_from_sells = 0
+    # cash flow : add new cash to your portfolio or withdraw 
+    print ("Do you want to add cash or withdraw money (y/n)?")
+    answer = input()
+    if answer == "y" :
+        print ("How much? Use negative number if you want to withdraw")
+        new_cash = float(input())
+        if new_cash > 0:
+            print ("You added " + str(new_cash) + " $.")
+            liquidity_from_sells = 0 + float(new_cash)
```

Comment:
Two comments:
- instead of re-defining `liquidity_from_sells` at every branch, you could define it before your new code (at line 223)
- to update an existing numerical variable you can do something like this `liquidity_from_sells += new_cash`...also, no need to casting it to float again (`float(new_cash)`) since you already did it at line 228